README.md

```diff
@@ -1,2 +1,36 @@
-# aurelia-permissions
-Permission plugin that works with aurelia-auth.
+# aurelia-acl
+
+[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg?maxAge=2592000?style=plastic)](https://gitter.im/SpoonX/Dev)
+
+> Permission plugin that works with aurelia-auth.
```

Comment:
authentication. Also make it a [link to authentication](https://github.com/SpoonX/aurelia-authentication)